tests/cli-self-upd.rs

```diff
@@ -900,19 +690,10 @@ fn reinstall_specifying_different_toolchain() {
 }
 
 #[test]
-#[cfg(unix)]
-fn produces_env_file_on_unix() {
+fn install_sets_up_stable() {
```

Comment:
this seems to be coming from nowhere - I can't see this test removed elsewhere in the diff.

Reply:
...I have no idea. I'm writing this off as "git did a badmerge somehow".